Doc/library/asyncio-sync.rst

```diff
@@ -356,6 +357,135 @@ BoundedSemaphore
       :ref:`What's New in 3.10's Removed section <whatsnew310-removed>`
       for more information.
 
+
+Barrier
+=======
+
+   .. versionadded:: 3.10
```

Comment:
It's too late to push this to 3.10, but 3.11 is OK.

Reply:
Ok, I will change